qa-tests-backend/src/test/groovy/UpgradesTest.groovy

```diff
@@ -200,7 +200,7 @@ class UpgradesTest extends BaseSpecification {
     }
 
     @Tag("Upgrade")
-    @IgnoreIf({ Env.CI_JOB_NAME.contains("postgres") })
+    // @IgnoreIf({ Env.CI_JOBNAME.contains("postgres") })
```

Comment:
remove?

Reply:
Yeah trying it out before remove, CI will tell us the truth.